tests/string/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -85,14 +87,14 @@ $string = "foo\\\\\\\\bar";
 $string = "foo\\"bar";
 $string = "foo\\"";
 $string = "foo'bar";
-$string = "foo\\377bar";
-$string = "foo\\3";
-$string = "foo\\xFFbar";
-$string = "foo\\xr";
-$string = "foo\\x";
-$string = "foo\\u{ff}bar";
-$string = "foo\\ubar";
-$string = "foo\\u";
+$string = "foo\\\\377bar";
```

Comment:
Broke :disappointed:

Reply:
Yup my bad, didn't realize what these were - just put back!

Reply:
@mgrip do not need to admit mistakes, it's okay search better solution